src/MudBlazor/Styles/components/_select.scss

```
@@ -128,3 +128,8 @@
 .mud-width-content {
     max-width: min-content;
 }
+
+.mud-select-placeholder {
+    color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)